core/controllers/acl_decorators.py

```diff
@@ -2927,6 +2927,35 @@ def test_can_access_topics_and_skills_dashboard(self, **kwargs):
     return test_can_access_topics_and_skills_dashboard
 
 
+def can_access_topics_and_skills_dashboard_read_only(handler):
+    """Decorator to check whether the user can access the topics and skills
+    dashboard in read only manner.
```

Comment:
Doesn't look like we are doing any checks here. It looks identical to `open_access` acl decorator.

Hm, not sure if we should make this an "open access" end point actually. @seanlip any thoughts as codeowner?

Reply:
I agree that this seems weird, and it might be due to a misinterpretation of the wording of the original issue (see point 3 of https://github.com/oppia/oppia/issues/14845). @soumyo123-prog -- the idea is that the exploration editor page can be generally accessed in read-only fashion. This does not entail making the topics-and-skills-dashboard page generally accessible.

What I would suggest is figuring out what exactly the exploration editor needs from the backend, and making a separate backend handler to supply that data (and only that data) that has the right permissions.

Reply:
@kevintab95 @seanlip PTAL. Added the new handler to provide only the categorised and untriaged skills data which is required by the exploration editor page. I made this handler open_access because any user (either logged in or not) should be able to see the data in read only fashion. Updating the skills in the exploration editor is still confined to users with proper rights. Also, updated the `After` video.